src/pages/index.tsx

```diff
@@ -74,6 +74,7 @@ import Image from "next/image";
 import { Loading, LoadingPage } from "~/components/loading";
 import { useState } from "react";
 import toast from "react-hot-toast";
```

Comment:
ngl bro not digging imports in the middle of the page... specially since next image is imported down here but also used above... kinda sus